.github/workflows/release.yml

```diff
@@ -26,38 +26,44 @@ jobs:
         id: codesign
         env:
           VERSION: v0
-
-      - uses: hashicorp/setup-hc-releases@v1
+      - name: Setup hc-releases
+        uses: hashicorp/actions-setup-hc-releases@v2
```

Comment:
this should be `setup-hc-releases@v2`, we can drop `actions-`

Reply:
thank you for doing this! ❤️